### ISSUE TITLE
Describe the command line syntax of --submission_information 

### ISSUE THREAD

**Issue by DerGary** [fastlane/fastlane#16674]
### Question Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Question Subject
Describe the command line syntax of --submission_information 

#### Question Description
I don't unterstand how to provide the value of --submission_information from the command line.
I already searched here: 
- https://docs.fastlane.tools/actions/deliver/#submission_information
- https://github.com/fastlane/fastlane/issues/13730
but both do not show an example of how it should look or how the syntax is.
In the documentation it says the value has to be a hash but not how the syntax of a hash looks like.
I tried it like this: 

````
--submission_information { export_compliance_encryption_updated: false, export_compliance_uses_encryption: false, content_rights_contains_third_party_content: false, add_id_info_uses_idfa: false }

--submission_information "{ export_compliance_encryption_updated: false, export_compliance_uses_encryption: false, content_rights_contains_third_party_content: false, add_id_info_uses_idfa: false }"

--submission_information "({ export_compliance_encryption_updated: false, export_compliance_uses_encryption: false, content_rights_contains_third_party_content: false, add_id_info_uses_idfa: false })"

--submission_information "({ export_compliance_encryption_updated: false, export_compliance_uses_encryption: false, content_rights_contains_third_party_content: false, add_id_info_uses_idfa: false })"

--submission_information {export_compliance_encryption_updated:false,export_compliance_uses_encryption:false,content_rights_contains_third_party_content:false,add_id_info_uses_idfa:false}
````

I always get this error message:
```
[!] 'submission_information' value must be a Hash! Found String instead.
##[error]Error: fastlane failed with return code: 1
```

**Comment by DerGary** [comment]
<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.15.4                                     |
| Ruby                        | 2.7.0                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.24.3 (Apple Git-128)          |
| Installation Source         | /usr/local/lib/ruby/gems/2.7.0/bin/fastlane |
| Host                        | Mac OS X 10.15.4 (19E287)                   |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.7.0/lib            |
| OpenSSL Version             | OpenSSL 1.1.1d  10 Sep 2019                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 11.5                                        |


### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |


### fastlane files:

**No Fastfile found**


**No Appfile found**


### fastlane gems

| Gem      | Version     | Update-Status |
| -------- | ----------- | ------------- |
| fastlane | 2.150.0.rc1 | ✅ Up-To-Date  |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.4.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| CFPropertyList            | 3.0.2        |
| claide                    | 1.0.3        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.15.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.5.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.7.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.7.1        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| babosa                    | 1.0.3        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.73.0       |
| faraday                   | 0.17.3       |
| unf_ext                   | 0.0.7.6      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.13.1       |
| fastimage                 | 2.1.7        |
| gh_inspector              | 1.1.3        |
| json                      | 2.3.0        |
| mini_magick               | 4.10.1       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.3.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.7.5        |
| bundler                   | 2.1.2        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| jwt                       | 2.1.0        |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mini_mime                 | 1.0.2        |
| multi_json                | 1.14.1       |
| signet                    | 0.13.0       |
| memoist                   | 0.16.2       |
| os                        | 1.0.1        |
| googleauth                | 0.11.0       |
| httpclient                | 2.8.3        |
| google-api-client         | 0.38.0       |
| google-cloud-env          | 1.3.1        |
| google-cloud-errors       | 1.0.0        |
| google-cloud-core         | 1.5.0        |
| digest-crc                | 0.5.1        |
| google-cloud-storage      | 1.25.1       |
| emoji_regex               | 1.0.1        |
| jmespath                  | 1.4.0        |
| aws-partitions            | 1.333.0      |
| aws-eventstream           | 1.0.3        |
| aws-sigv4                 | 1.1.1        |
| aws-sdk-core              | 3.101.0      |
| aws-sdk-kms               | 1.35.0       |
| aws-sdk-s3                | 1.70.0       |
| uri                       | 0.10.0       |
| forwardable               | 1.3.1        |
| logger                    | 1.4.2        |
| cgi                       | 0.1.0        |
| timeout                   | 0.1.0        |
| stringio                  | 0.1.0        |
| ipaddr                    | 1.2.2        |
| openssl                   | 2.1.2        |
| ostruct                   | 0.2.0        |
| strscan                   | 1.0.3        |
| date                      | 3.0.0        |
| delegate                  | 0.1.0        |
| fileutils                 | 1.4.1        |
| io-console                | 0.5.3        |
| zlib                      | 1.1.0        |
| singleton                 | 0.1.0        |
| rexml                     | 3.2.3        |
| open3                     | 0.1.0        |
| yaml                      | 0.1.0        |
| psych                     | 3.1.0        |
| mutex_m                   | 0.1.0        |
| webrick                   | 1.6.0        |
</details>


*generated on:* **2020-06-25**
</details>

**Comment by janpio** [comment]
I have _no_ idea, but I would try `--submission_information[export_compliance_encryption_updated]=false`.

**Comment by markmuniz** [comment]
I'm having issues getting this to work via command line as well. I've also noticed that `--submission_information` no longer appears to be an optional parameter as of `2.150.0.RC1`

**Comment by joshdholtz** [comment]
@markmuniz Thanks for referencing! It also took me a while to figure it out to be honest. I'll make sure it gets added to the docs 💪

**Comment by DerGary** [comment]
I will try this, the next time I need to deploy an app. I will report back if it works. Thanks for updating the docs (I think it will reflect the changes after 2.150.0 final will be released, correct?)

**Comment by joshdholtz** [comment]
Correct correct! Which _SHOULD_ be today 😊

**Comment by joshdholtz** [comment]
Closing this since this should be fixed now! Latest version is `2.150.1` and the docs are live!
